sonar-db/src/main/java/org/sonar/db/measure/MeasureMapper.java

```diff
@@ -19,15 +19,20 @@
  */
 package org.sonar.db.measure;
 
+import java.util.Collection;
 import java.util.List;
 import org.apache.ibatis.annotations.Param;
+import org.apache.ibatis.session.ResultHandler;
 
 public interface MeasureMapper {
 
   List<MeasureDto> selectByQuery(@Param("query") MeasureQuery query);
 
+  void selectByQuery(@Param("query") MeasureQuery query, ResultHandler resultHandler);
+
   List<PastMeasureDto> selectPastMeasures(@Param("componentUuid") String componentUuid, @Param("analysisUuid") String analysisUuid, @Param("metricIds") List<Integer> metricIds);
 
-  void insert(MeasureDto measureDto);
+  List<MeasureDto> selectProjectMeasuresOfDeveloper(@Param("developerId") long developerId, @Param("metricIds") Collection<Integer> metricIds);
```

Comment:
the "project" in the name matter since we only care about measure for projects components here (qualifier=TRK)